components/scream/tests/coupled/homme_physics/CMakeLists.txt

```diff
@@ -0,0 +1,99 @@
+include (ScreamUtils)
+
+# Needed for RRTMGP
+find_library(NETCDF_C netcdf HINTS ${NetCDF_C_PATHS}/lib)
+## Add CUDA flags for YAKL
+if (CUDA_BUILD)
+    set(ARCH "CUDA")
+    set(YAKL_CXX_FLAGS "-D__USE_CUDA__ --expt-extended-lambda --expt-relaxed-constexpr ${CUDA_FLAGS} ${YAKL_CXX_FLAGS}")
+endif()
+
+# Get or create the dynamics lib
+#                 HOMME_TARGET   NP PLEV QSIZE_D
+CreateDynamicsLib("theta-l_kokkos"  4   72   35)
+
+set (NEED_LIBS cld_fraction shoc p3 scream_rrtmgp rrtmgp ${NETCDF_C} ${dynLibName} scream_control scream_share physics_share yakl)
+
+# Add source files
+set (SRC shoc_cld_p3_rrtmgp.cpp ${SCREAM_BASE_DIR}/src/physics/rrtmgp/tests/rrtmgp_test_utils.cpp)
+
+# Test Homme standalone
+if (NOT SCREAM_AUTOTESTER AND NOT EKAT_ENABLE_VALGRIND)
+  CreateUnitTest(homme_physics_stand_alone "homme_physics_stand_alone.cpp" "${NEED_LIBS}" EXE_ARGS "< namelist.nl" LABELS "dynamics;driver;shoc;cld;p3;rrtmgp;physics")
```

Comment:
This might be just me, but `standalone` seems inappropriate if the test has 2+ processes. You probably intended it as "not through cime", but all the cmake tests we have are "not through cime", so we'd have to add it to all the tests...

Some possible alternative names: `homme_scream_physics` or `homme_p3_shoc_cld_rrtmgp`.

Reply:
Yes, that's a good point.  I'm down for removing "stand_alone" from all of the "coupled" tests and just making the name fit all of the procs in it.  So in this case `homme_p3_shoc_cld_rrtmgp` would be the most descriptive.